### DISCUSSION TITLE
running oobabooga in a docker container

### DISCUSSION THREAD

**Discussion by Kapowpenguin1** [oobabooga/text-generation-webui#3463] [Q&A]
i've been trying to run oobabooga in a docker container without using the guide and i'm having trouble accessing the webui. the furthest i've been able to get is firefox complaining that the connection was "reset". curl complains about an "Empty reply from server" but telnet seems to connect properly. i configured a container using nvidia's cuda container to run the one click linux installer, if that helps

nevermind, i decided to just suck it up and go through the guide

**Comment by jtara1** [comment]
Looks like this is the guide for reference https://github.com/oobabooga/text-generation-webui/wiki/09-%E2%80%90-Docker
